src/components/balances/AssetsTable/index.tsx

```diff
@@ -93,10 +92,8 @@ const SendButton = ({
   tokenInfo: TokenInfo
   onClick: (tokenAddress: string) => void
 }): ReactElement => {
-  const spendingLimit = useSpendingLimit(tokenInfo)
-
   return (
-    <CheckWallet allowSpendingLimit={!!spendingLimit}>
```

Comment:
It's OK to just not enable this button for non-owners.

Reply:
It's possible to add spending limits for non-owners though.

Reply:
I'm aware but this button is not worth the extra requests. They can still transact via the New tx button.